packages/overlay/test/index.ts

```diff
@@ -760,6 +760,79 @@ export const runOverlayTriggerTests = (type: string): void => {
                 document.body.removeChild(longContent);
             });
         });
+        describe('receives-focus behavior', () => {
+            it('focuses first focusable element when modal overlay opens with Enter and receives-focus is not false but auto by default', async () => {
+                const el = await fixture<OverlayTrigger>(html`
+                    <overlay-trigger type="modal">
```

Comment:
```suggestion
                    <overlay-trigger type="modal" triggered-by="click">
```

This needs to be included to avoid browser logs firing the console warning